src/amo/components/Link.js

```diff
@@ -18,7 +19,20 @@ export class LinkBase extends React.Component {
   }
 
   render() {
-    const { base, children, to } = this.props;
+    const { base, children, href, to, ...customProps } = this.props;
+
+    if (
+      typeof href === 'string' && href.startsWith('/') &&
```

Comment:
Looks like you can remove this check since it's not used in the if statement below anymore: `href.startsWith('/')`